Source/Core/Core/AchievementManager.cpp

```diff
@@ -1559,14 +1573,36 @@ void AchievementManager::HandleLeaderboardStartedEvent(const rc_runtime_event_t*
     {
       OSD::AddMessage(fmt::format("Attempting leaderboard: {}", m_game_data.leaderboards[ix].title),
                       OSD::Duration::VERY_LONG, OSD::Color::GREEN);
+      LeaderboardDisplay display{.id = runtime_event->id,
+                                 .format = m_game_data.leaderboards[ix].format};
+      rc_runtime_format_lboard_value(display.value.data(), FORMAT_SIZE, runtime_event->value,
+                                     display.format);
+      m_active_leaderboards.push_back(display);
       return;
     }
   }
   ERROR_LOG_FMT(ACHIEVEMENTS, "Invalid leaderboard started event with id {}.", runtime_event->id);
 }
 
+void AchievementManager::HandleLeaderboardUpdatedEvent(const rc_runtime_event_t* runtime_event)
+{
+  for (LeaderboardDisplay& display : m_active_leaderboards)
```

Comment:
```suggestion
  for (const auto& display : m_active_leaderboards)
```

Reply:
I'm probably wrong on this but I don't think I can make this a const because when I pass display.value.data() to the format method, if display is const then data() is const and can't be modified.